cat-theme/src/css/abstracts/_variables.scss

```diff
@@ -0,0 +1,26 @@
+// Colors
+$color-body: #f9f9f9;
+$color-white: #fff;
```

Comment:
Rather than defining a variable by the color, it would be better to define the variable by it's usage to give it more flexibility.